ES/ESmy001.xml

```diff
@@ -363,129 +751,143 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                                  <item xml:id="div5"> Sentence dividers are accompanied by the
                                     paragraph signs, on the left from the text column and between
-                                    the two columns (used throughout the Ms.).</item>
-                                 <item xml:id="div6"> III. Doubled nine dot asterisks, with lines in
+                                    the two columns (used throughout).</item>
+                                 <item xml:id="div6">Doubled nine dot asterisks, with lines in
                                     between (esp. fols.: 21rb, l.31; 21va, l.8; 124ra, ls. 8, 19;
-                                    144va, l.22; 144vb, l.25);</item>
-                                 <item xml:id="div7"> nine dot asterisks, connected with lines (esp.
-                                       <locus target="#126rb">126rb</locus>, l.8);</item>
-                                 <item xml:id="div8"> four dot asterisks, connected with lines (esp.
-                                    fols.: 131vb, l.29; 135vb, l.17);</item>
-                                 <item xml:id="div9"> Chain of black lines and red and black dots
+                                    144va, l.22; 144vb, l.25).</item>
+                                 <item xml:id="div7">Nine dot asterisks, connected with lines (esp.
+                                       <locus target="#126rb">126rb</locus>, l.8).</item>
+                                 <item xml:id="div8">Four dot asterisks, connected with lines (esp.
+                                    fols.: 131vb, l.29; 135vb, l.17).</item>
+                                 <item xml:id="div9">Chain of black lines and red and black dots
                                     (e.g. <locus target="#1ra">1ra</locus>b, l.1; 1ra, <locus from="16" to="17">16-17</locus>; 12rb, <locus target="#20">20</locus>, 28; 15rb, <locus target="#15">15</locus>; 16vb,
                                     l.31; 17ra, l.1; 18vb, l.7; 19vb, l.21; 53rb, l.11; 63vb, l.1;
                                     89ra, <locus from="30" to="31">30-31</locus>; 89rb, l.1; 109vb,
-                                    l.1; 126rb, l.8; 150rb, l.25);</item>
-                                 <item xml:id="div10"> Crux ansata with coronis, marking the
-                                    beginning of each new text (used thoughout the Ms.).</item>
+                                    l.1; 126rb, l.8; 150rb, l.25).</item>
+                                 <item xml:id="div10">Crux ansata with coronis, marking the
+                                    beginning of each new text (used thoughout).</item>
                               </list>
                            </ab>
-                           <ab type="CruxAnsata"> </ab>
-                           <ab type="coronis"> </ab>
+                           <ab type="CruxAnsata"/>
```

Comment:
are the elements present? if yes maybe better add locus?

Reply:
done